src/helper.js

```diff
@@ -125,11 +125,11 @@ class helper {
 			return url;
 	}
 
-	static httpToWsUrl(str) {
+	static httpToWssUrl(str) {
 		let url = new Url(str);
 
 		if (this.validURL(url)) {
-			url.set('protocol', 'ws:');
```

Comment:
will it still support non https nodes?

Reply:
No, the default node should all be in HTTPS, right?
But let me just made a port checking, so it can switch between `wss` or `ws`.